x-pack/plugins/observability_solution/infra/public/components/asset_details/charts/container_charts.tsx

```diff
@@ -0,0 +1,171 @@
+} from '../hooks/use_container_metrics_charts';
+import { Section } from '../components/section';
+import { ChartsGrid } from '../charts_grid/charts_grid';
+import { Chart } from './chart';
+import { useIntegrationCheck } from '../hooks/use_integration_check';
+import { TitleWithTooltip } from '../components/section_title';
+import { CONTAINER_METRIC_GROUP_TITLES } from '../translations';
+import { CONTAINER_METRICS_DOC_HREF } from '../../../common/visualizations/constants';
+import { INTEGRATIONS } from '../constants';
+import { MetricsChartsFields, ContainerMetricTypes } from './types';
+
+interface Props extends MetricsChartsFields {
+  metric: ContainerMetricTypes;
+}
+
+const FRAGMENT_BASE = 'key-metrics';
+
+export const DockerCharts = React.forwardRef<HTMLDivElement, Props>(
+  ({ assetId, dataView, dateRange, metric }, ref) => {
+    const { charts } = useDockerContainerPageViewMetricsCharts({
+      metric,
+      metricsDataViewId: dataView?.id,
+    });
+    return (
+      <Section
+        title={
+          <TitleWithTooltip
+            title={CONTAINER_METRIC_GROUP_TITLES[metric]}
+            tooltipContent={
+              <EuiText size="xs">
+                <FormattedMessage
+                  id="xpack.infra.assetDetails.charts.container.toolTip"
+                  defaultMessage="See container-related {link} for more information"
+                  values={{
+                    link: (
+                      <EuiLink
+                        data-test-subj="infraAssetDetailsViewContainerMetricsDocumentationLink"
+                        href={`${CONTAINER_METRICS_DOC_HREF}#${FRAGMENT_BASE}-${metric}`}
+                        target="_blank"
+                        className={cx({
+                          [css`
+                            text-transform: lowercase;
+                          `]: metric !== 'cpu',
+                        })}
+                      >
+                        <FormattedMessage
+                          id="xpack.infra.assetDetails.charts.container.toolTip.linkText"
+                          defaultMessage="{metric} metrics"
+                          values={{ metric: CONTAINER_METRIC_GROUP_TITLES[metric] }}
+                        />
+                      </EuiLink>
+                    ),
+                  }}
+                />
+              </EuiText>
+            }
+          />
+        }
+        data-test-subj="infraAssetDetailsDockerContainerChartsSection${metric}"
+        id="dockerContainerCharts"
+        ref={ref}
+      >
+        <ChartsGrid columns={2}>
+          {charts.map((chart) => (
+            <Chart
+              key={chart.id}
+              {...chart}
+              assetId={assetId}
+              dateRange={dateRange}
+              queryField={findInventoryFields('container').id}
+            />
+          ))}
+        </ChartsGrid>
+      </Section>
+    );
+  }
+);
+
+export const KubernetesCharts = React.forwardRef<HTMLDivElement, Props>(
```

Comment:
How about moving this to `kubernetes_charts`? Renaming this component to `KubernetesContainerCharts`? In the `kubernetes_charts` there is already a `KubernetesCharts` but we can rename it to `KubernetesNodeCharts`